src/docs/Glossary.adoc

```diff
@@ -33,37 +33,57 @@ Scheduler Web Interface:: ProActive component that provides a web interface to t
+
+[[_glossary_analytics]]
+Analytics::  ProActive service responsible to gather monitoring information for <<_glossary_job,*Jobs*>> and <<_glossary_proactive_node,*ProActive Nodes*>>
+
 [[_glossary_job_analytics]]
-Job Analytics:: ProActive component that provides an overview of executed workflows along with their input variables and results.
+Job Analytics:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of executed workflows along with their input variables and results.
+
+[[_glossary_node_gantt]]
+Node Gantt:: ProActive component, part of the <<_glossary_analytics,*Analytics*>> service, that provides an overview of <<_glossary_proactive_node,*ProActive Nodes*>> usage over time.
+
+[[_glossary_analytics_portal]]
+Analytics Portal:: Web interface of the <<_glossary_job_analytics,*Analytics*>> component.
 
 [[_glossary_notification_service]]
 Notification Service:: ProActive component that allows a user or a group of users to subscribe to different types of notifications (web, email, sms or a third-party notification) when certain event(s) occur (e.g., job In-Error state, job in Finished state, scheduler in Paused state, etc.).
 
 [[_glossary_notification_portal]]
-Notification Portal::
-ProActive portal to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
+Notification Portal:: Web interface to visualize notifications generated by the <<_glossary_notification_service,*Notification Service*>> and manage subscriptions.
 
 [[_glossary_notification_subscription]]
-Notification Subscription::
-Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
+Notification Subscription:: Object of the <<_glossary_notification_service,*Notification Service*>> used to manage and configure notification reception.
 
 [[_glossary_notification_third_party]]
-Third Party Notification::
-Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
+Third Party Notification:: Notification method of the <<_glossary_notification_service,*Notification Service*>> that executes a script when a notification is triggered.
 
 [[_glossary_notification_methods]]
-Notification Method::
-Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
+Notification Method:: Component of the <<_glossary_notification_service,*Notification Service*>> that defines how the user is notified of notifications.
```

Comment:
how the user **receives** notifications

Reply:
remove `of notifications` and u might add method examples